azure-pipelines.yml

```diff
@@ -50,8 +46,7 @@ stages:
     - script: >-
         python -m pip install -r requirements/linter_requirements.txt
       displayName: 'Install tools'
-      # pip 21.1 emits a pile of garbage messages to annoy users :)
-      #      failOnStderr: true
+      continueOnError: true
```

Comment:
I don't understand. How can the lint job succeed if the tools are not installed?